### PR TITLE
Add sleep a few seconds between two continuous detach device operations

### PR DESCRIPTION
**Pull request by chunfuwen**
Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
legacy start libvirtd daemon IGNORED with function name: restart
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.multidisks.hotplug.multi_disks_test.disks_duplicate_target_same_bus_error_test

File: libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py [hunk 1]
```diff
@@ -1815,6 +1815,8 @@ def test_device_update_boot_order(disk_type, disk_path, error_msg):
                     dt_options = "--config"
                 ret = virsh.detach_disk(vm_name, device_targets[i],
                                         dt_options, **virsh_dargs)
+                #Sleep 3 seconds to let detach operation accomplish completely
+                time.sleep(3)
```
Comment (kylazhang) [inline libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py:1819]: I think we'd better add wait for in check_vm_partitions as the sleep time maybe various in different machine/arch

**Comment by chunfuwen** [comment]
paste new runtest result
************************************************
Command 'virsh list' finished with 0 after 0.3344709873199463s
[stdout] 
cleaning libvirtd logs...
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.multidisks.hotplug.multi_disks_test.disks_duplicate_target_same_bus_error_test

Requested MAC address release from persistent vm avocado-vt-vm1. Ignoring.
Test results available in /root/avocado/job-results/job-2020-04-01T11.48-36d8a30
Not logging /proc/pci (file does not exist)

File: libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py (hunk 1 above)
Comment (chunfuwen) [reply libvirt/tests/src/virtual_disks/virtual_disks_multidisks.py]: updated with add wait_for operation in case.

**Pull request merged**